src/objects/Opportunity.object

```diff
@@ -0,0 +1,403 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't seen any question that mentions editing Opportunity object. Could you please verify?